ci(agw): Filter lte/gateway/c code coverage

markjen opened the pull request:
Closes #6348

Remove coverage of generated code and included libraries.

Signed-off-by: Mark Jen <markjen@fb.com>

## Test Plan

Ran `make coverage` from /lte/gateway inside build docker instance and checked results using genhtml.

## Additional Information

Updated coverage report from my local build, showing reduced file set.

![after](https://user-images.githubusercontent.com/72896/123013131-3ea65300-d389-11eb-9b1e-98d0ecd1c1dc.png)

markjen committed:
Filter lte/gateway/c code coverage

Remove coverage of generated code and included libraries.

Signed-off-by: Mark Jen <markjen@fb.com>

themarwhal reviewed (approved):
oh yes I've been meaning to do this :) Thanks!

themarwhal commented:
Hmm a bit unfortunate that `lte/gateway/Makefile` isn't covered as one of the files that triggers the cpp code cov job: https://app.circleci.com/pipelines/github/magma/magma/26922/workflows/ac9330aa-b2dc-4837-a6e8-0c1529aa9395/jobs/289853

markjen committed:
Trigger cpp build on Makefile update

Add Makefile to list of paths that trigger cpp build in CI. Current list
misses Makefile changes because it lives in a parent dir of the source files.

Signed-off-by: Mark Jen <markjen@fb.com>

markjen commented:
rebased to try to trigger build

#ready2merge
